node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -181,33 +180,35 @@ class NodeRuntime(conf: Conf)(implicit scheduler: Scheduler) {
   implicit val nodeDiscoveryEffect: NodeDiscovery[Task] =
     new KademliaNodeDiscovery[Task](src, defaultTimeout)
 
+  if (Files.notExists(casperPersistencePath)) Files.createDirectories(casperPersistencePath)
+  val config =
+    coop.rchain.blockstorage.LMDBBlockStore.Config(casperPersistencePath, 1024 * 1024 * 1024)
+  val syncEffect: Sync[Effect] = SyncInstances.syncEffect
+  implicit val blockStore: BlockStore[Effect] =
+    LMDBBlockStore.create[Effect](config)(syncEffect, metricsEffect)
```

Comment:
not convinced this is a good spot for this init, but did not want to pollute the `Runtime`

Reply:
If you make the two changes above then this just becomes a one-liner similar to the other effect instantiations above, so then it should be fine.